src/main/java/frc/robot/RobotContainer.java

```diff
@@ -119,25 +121,28 @@ private void configureButtonBindings() {
     m_hood.setDefaultCommand(new ManualHood(() -> {
       if (m_operatorJoystick.getPOV() == 0 || m_operatorJoystick.getPOV() == 45 || m_operatorJoystick.getPOV() == 315) {
         return 0.6;
-      } else if (m_operatorJoystick.getPOV() == 180 || m_operatorJoystick.getPOV() == 135 || m_operatorJoystick.getPOV() == 225) {
+      } else if (m_operatorJoystick.getPOV() == 180 || m_operatorJoystick.getPOV() == 135
+          || m_operatorJoystick.getPOV() == 225) {
         return -0.6;
-      } 
+      }
       return 0.0;
     }, m_hood));
 
     m_turret.setDefaultCommand(new ManualTurret(() -> {
-      if (m_operatorJoystick.getPOV() == 90 || m_operatorJoystick.getPOV() == 45 || m_operatorJoystick.getPOV() == 135) {
+      if (m_operatorJoystick.getPOV() == 90 || m_operatorJoystick.getPOV() == 45
+          || m_operatorJoystick.getPOV() == 135) {
         return 0.3;
-      } else if (m_operatorJoystick.getPOV() == 270 || m_operatorJoystick.getPOV() == 225 || m_operatorJoystick.getPOV() == 315) {
+      } else if (m_operatorJoystick.getPOV() == 270 || m_operatorJoystick.getPOV() == 225
+          || m_operatorJoystick.getPOV() == 315) {
         return -0.3;
-      } 
+      }
       return 0.0;
     }, m_turret));
   }
 
   /**
-   * Resets encoders, gyro, any other sensors necessary for auto, etc...
-   * Also schedules autonomous command
+   * Resets encoders, gyro, any other sensors necessary for auto, etc... Also
+   * schedules autonomous command
```

Comment:
Please shorten:

`Resets sensors and schedules the autonomous command.`
or
`Zeroes sensors and schedules the autonomous command.`

Reply:
Resolved in bbd7d1a